lalrpop-test/src/generics_issue_417.lalrpop

```diff
@@ -0,0 +1,11 @@
+use generics_issue_104_lib::Generator;
+
+grammar<T> where T: Generator;
+
+extern {
+    type Error = String;
+}
+
+Id : String = <id:r"[a-zA-Z0-9]*"> => T::schema(id);
+
+pub Tuple : (String, String) = "(" <(<Id> "," <Id>)> ")";
```

Comment:
Nit: Missing newline at EOF (makes git slightly confused if text is appended in a later commit).

Reply:
Fixed